src/app/components/AssignmentDefaultRecurrenceDetails.tsx

```diff
@@ -15,7 +15,11 @@ export interface AssignmentDefaultRecurrenceDetailsProps {
 export default class AssignmentDefaultRecurrenceDetails extends React.PureComponent<AssignmentDefaultRecurrenceDetailsProps, any> {
     render() {
         const { data: { days, startTime, endTime, sheriffsRequired } } = this.props;
-        const dayDisplay = EnumUtils.displayEnum(DaysOfWeek, days).join(", ");
+        
+        let dayDisplay = EnumUtils.displayEnum(DaysOfWeek, days).join(", ");
```

Comment:
Can we add some of these cases to the unit tests for this utility?  Or is it very specific to the days of the week?

Reply:
I think it is pretty specific to the days of the week

Reply:
hmm, in that case.  We may want to wrap the enumUtils in a function specific for days of the week and then write tests for it.

Reply:
okay sound good

Reply:
A little trick in typescript, you can make functions look like static members of enums by using namespaces.  i.e.:

```ts
export enum DaysOfWeek {
    Mon = 1 << 0,
    Tue = 1 << 1,
    Wed = 1 << 2,
    Thu = 1 << 3,
    Fri = 1 << 4,
    Sat = 1 << 5,
    Sun = 1 << 6,
    Everyday = Mon | Tue | Wed | Thu | Fri | Sat | Sun,
    Weekdays = Mon | Tue | Wed | Thu | Fri
}

export namespace DaysOfWeek{
    export function display(value:DaysOfWeek):string{
        // Display logic here
    }
}

const test:DaysOfWeek = DaysOfWeek.Sat;
DaysOfWeek.display(test);
```